src/GHCMod.hs

```diff
@@ -256,6 +257,14 @@ globalArgSpec =
                reqArg "OPT" $ \g o ->
                    o { ghcUserOptions = g : ghcUserOptions o }
 
+      , option "" ["map-file"] "Redirect one file to another, --map-file \"file1.hs=file2.hs\"" $
```

Comment:
Please add that in the actual source because iirc github discards these comments (or rather makes them super hard to find) once you push over this changeset.